RFC/src/RFC-0241_AtomicSwapXMR.md

```diff
@@ -0,0 +1,766 @@
+
+* Negotiation - Here, both parties negotiate about the values and how the Monero and Tari Utxo's will look
+* Commitment - Here, both parties commit to their use of keys, and Bob commits to the refund transaction
+* XTR payment - Here, the XTR payment is made to a multi-party UTXO containing a script
+* XMR Payment - The Monero payment is made to a multiparty [scriptless script](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts) UTXO.
+* Claim XTR - Here, the XTR is claimed, and in claiming it, the XMR private key is revealed
+* Claim XMR - Here, the XMR is claimed with the revealed key.
+
+Please take note of the notation used in [TariScript] and specifically notation used on the signatures on the [transaction inputs](RFC-0201_TariScript.md#transaction-input-changes) and on the signatures on the [transaction outputs](RFC-0201_TariScript.md#transaction-output-changes), other notation will be noted in the [Notation](#notation) section.
+
+
+### Alternative approach TL;DR
+
+This scheme revolves around Alice, who wants to exchange some of her Tari for Bob's Monero. Because they don't 
+trust each other, they have to commit some values to do the exchange. And if something goes wrong here, we want to ensure
+that we can refund both parties either in Monero or Tari.
+
+How this works is that Alice and Bob create a shared output on both chains. The Monero output is a simple aggregate key
+to unlock the UTXO, while the Tari UTXO is unlocked by either one of two keys. The block height determines the unlock key on Tari. 
+
+The TariScript will decide the unlock key to claim the Tari amount. The script will require the user to input their part
+of the Monero aggregate key used to lock the Monero UTXO. The script can end one of three ways, one for the happy path if
+Alice gives Bob the pre_image after she checked and verified the Monero UTXO, one for her to reclaim her Tari amount if Bob
+disappears or tries to break the contract. And lastly, one for Bob to claim the Tari if Alice disappears after he publishes the
+Monero UTXO.
+
+### Alternative approach TariScript
+
+The Script used for the Tari UTXO is as follows:
+``` TariScript,ignore
+   CheckHeight(height_1)
+   LtZero
+   IFTHEN
+      HashSha256 
+      PushHash(HASH256{pre_image})
+      EqualVerify
+      Ristretto
+      PushPubkey(X_b)
+      EqualVerify
+      PushPubkey(K_{Sb})
+   Else
+      CheckHeight(height_2)
+      LtZero
+      IFTHEN
+         Ristretto
+         PushPubkey(X_a)
+         EqualVerify
+         PushPubkey(K_{Sa})
+      Else
+         Ristretto
+         PushPubkey(X_b)
+         EqualVerify
+         PushPubkey(K_{Sb})
+      ENDIF
+   ENDIF
+```
+
+Here `height_1` is the lock height till Alice can claim the transaction. If Alice fails to publish the refund transaction
+after `height_2,` Bob can claim the lapse transaction.
+
+### Alternative approach Negotiation
```

Comment:
All these titles are weird in terms of capitalization. Remove the `Alternative Approach part of all these headings. The section heading is clear